generators/server/templates/gradle/_yeoman.gradle

```diff
@@ -1,12 +1,20 @@
-import org.apache.tools.ant.taskdefs.condition.Os
-apply plugin: 'com.moowork.gulp'
+apply plugin: 'com.moowork.node'
 
-task bower(type: Exec) {
-    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-        commandLine 'cmd', '/c', 'bower', 'install'
-    }else{
-        commandLine 'bower', 'install'
-    }
+task bower(type: NodeTask) {
+    description = "Installs dependencies using Bower"
+    script = file("${project.projectDir}/node_modules/bower/bin/bower")
```

Comment:
ya but the I saw somebody added a os check on our new webpack task on one of the gradle file, were based on os the bin path was different